docs/developer/concepts.ipynb

```diff
@@ -55,8 +55,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "assert sc.is_equal(ds['a']['x', 0:1], ds['x', 0:1]['a'])\n",
-    "assert sc.is_equal(ds['a']['x', 0], ds['x', 0]['a'])"
+    "assert sc.isequal(ds['a']['x', 0:1], ds['x', 0:1]['a'])\n",
+    "assert sc.isequal(ds['a']['x', 0], ds['x', 0]['a'])"
```

Comment:
Wondering if this should be named something like `allequal`? Similar to `np.allclose`? It may be confusing that, e.g., `isclose` returns a variable, but `isequal` returns `bool`?

Reply:
I think you're right, though `allequal` doesn't seem so clean when applied to `DataArray` and `DataSet`, though I think it would have worked fine for `Variable`.

Maybe "match" or "same"?